templates/check-java.sh.erb

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+# This file is managed by Puppet
+#
+# This file should be identical to 'check-java.sh' in the Jira 7.0.4 standalone
+# tarball, with 2 exceptions: 'grep "java version"' has been changed
+# to 'grep "version"' in order to allow JIRA to run on OpenJDK, and the
```

Comment:
Understood.

I've updated the change to use a vanilla `check-java.sh` script. :slightly_smiling_face: